views/courses/edit-course.hbs

```diff
@@ -301,7 +301,7 @@
 					</button>
 
                     {{> "lib/components/csrfInput"}}
-					<button type="submit" class="btn btn-primary btn-submit">
+					<button type="submit" class="btn btn-primary btn-submit" data-testid="deleteModalDeleteButton">
```

Comment:
see above

Reply:
changed to modal_delete_course_button, too